pkg/packagekit/autoupdates.jsx

```diff
@@ -337,6 +336,142 @@ class Dnf5Impl extends ImplBase {
     }
 }
 
+class AptImpl extends ImplBase {
+    // Define Origins-Patterns for different upgrade types
+    originsPatterns = {
+        all: '"origin=*";',
+        stable: '"origin=Debian,codename=${distro_codename},label=Debian";',
+        security: '"origin=Debian,codename=${distro_codename}-security,label=Debian-Security";',
+        securityAlt: '"origin=Debian,codename=${distro_codename},label=Debian-Security";'
+    };
+
+    async getConfig() {
+        this.packageName = "unattended-upgrades";
+
+        try {
+            await cockpit.spawn(["dpkg", "-l", this.packageName], { err: "ignore" });
+            this.installed = true;
+        } catch (ex) {
+            this.installed = false;
+            debug("apt getConfig: not installed:", ex);
+            return;
+        }
+
+        try {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22515-acb2e3fc-20251125-163641-fedora-43-devel/Coverage/lcov/github-pr.diff.gcov.html)